### PR TITLE
Add Donny!

### PR DESCRIPTION
**Pull request by paulcpederson**
@emily-plummer 

Tried to execute my vision and added some donny illustrations. I also added some procedurally generated sprinkles and gave the whole thing a nice face lift.

I moved pages to their own folder so we can use urls like `/speak/` instead of `speak.html`. Let me know if that is going to break anything?

### CODE REVIEW

File: js/sprinkles.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  var colors = [
+    'rgba(239, 229, 120, .6)', // yellow
+    'rgba(183, 229, 207, .6)', // mint
+    'rgba(248, 202, 205, .6)', // pink
+    'rgba(245, 84, 137, .6)',  // magenta
+    'rgba(255, 255, 255, .6)'  // white
+  ]
+
+  // Generate array of sprinkles
+  var MAX = 60
```
Comment (visnup) [inline js/sprinkles.js]: NEEDS MOAR SPRINKLES

**Comment by paulcpederson** [comment]
@emily-plummer ok I just took the content from the issues for speaking and performing and dropped them in the pages at `/speak/` and `/perform/` as well. (They also have links to the GitHub issues). Now the whole site is hecka shiny. :+1:

**Comment by emily-plummer** [comment]
looks awesome!!!! thanks so much!

**Comment by visnup** [comment]
RAINING SPRINKLES!!!!!

**Pull request merged**